scripts/easy-secrets-gen.sh

```diff
@@ -2,7 +2,22 @@
 
 cd $(git rev-parse --show-toplevel)
 
-DEST_DIR=${1:-.}
+DEST_DIR=${1:-components/01-secrets/}
+
+if ! type -p kubeseal kubectl > /dev/null; then
+    echo "You must have kubeseal & kubectl installed to use this script" >&2
```

Comment:
You also need `kubeseal` able to talk to your cluster as well.

Reply:
Do we really want to overload the script with context handling? Or is there some other thing you were thinking?

Reply:
this should be handled with:

https://github.com/rackerlabs/understack/pull/117/files#diff-e2a91e11ec7b4c9466e228236ba28b9fa43e645cbb064eb86d3838f014d41861R12-R15